### DISCUSSION TITLE
Node packages cannot be imported in the main thread using the import syntax

### DISCUSSION THREAD

**Discussion by chunfenghuayu1** [electron-vite/electron-vite-vue#338] [General]
for example：
if i use express/express-http-proxy in main thread
`const express = require('express')
const expressProxy = require('express-http-proxy')`
it will be normal。

but when i use import syntax
`import express from 'express'
import expressProxy from 'express-http-proxy'`
it will output a error：
**TypeError: require$$0$2 is not a functionat D:\code**

if it my project problem，how i fix it？

**Comment by chunfenghuayu1** [comment]
i cannot understand why electron can use import，but express cannot？

**Comment by caoxiemeihao** [comment]
This may be a problem caused by `esbuild` 👀

---

But. You can move `express` and `express-http-proxy` to `dependencies` to resolve it.

**Reply by chunfenghuayu1** [reply]
> This may be a problem caused by `esbuild` 👀
> 
> But. You can move `express` and `express-http-proxy` to `dependencies` to resolve it.

thank you ， i fixed it with this option。
